ruby/vampires.rb

```diff
@@ -70,6 +71,8 @@
 		vampire_status = "Definitly a vampire."
 	elsif (possibility_3 == true) && (name == "Drake Cula" || name == "Tu Fang")
 		vampire_status = "Definitely a vampire."
+	elsif (possibility_4 == true) && (name == "Drake Cula" || name == "Tu Fang")
```

Comment:
I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```